examples/maps_account/101-azure-maps-account/configuration.tfvars

```
@@ -0,0 +1,20 @@
+global_settings = {
+  default_region = "region1"
+  regions = {
+    region1 = "australiaeast"
+  }
+}
+
+resource_groups = {
+  test = {
+    name = "rg-maps-test"
+  }
+}
+
+maps_account = {
```

```suggestion
maps_accounts = {
```

Suggest you use the plural version for top variables